### PR TITLE
[AAE-4697] Restrict content node selector breadcrumb when destination folder path is wrong

### PR DESCRIPTION
**Pull request by sivakumar414ram**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [x] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

https://alfresco.atlassian.net/browse/AAE-4697

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Review by mauriziovitale — APPROVED** [review]

File: lib/process-services-cloud/src/lib/form/services/content-cloud-node-selector.service.ts
Comment (arditdomi) [inline lib/process-services-cloud/src/lib/form/services/content-cloud-node-selector.service.ts:64]: fetchNodeIdFromRelativePath and fetchAliasNodeId are the same methods, the only difference is the opts parameter which you can make optional and keep just one. What do you think?

Comment (arditdomi) [reply lib/process-services-cloud/src/lib/form/services/content-cloud-node-selector.service.ts:64]: Maybe having has it's benefits as from the names you understand the flow

Comment (sivakumar414ram) [reply lib/process-services-cloud/src/lib/form/services/content-cloud-node-selector.service.ts:64]: yup, In the first place i had single method but it's little hard to understand the flow. May we can have more generic name so that we can have optional param.

File: lib/process-services-cloud/src/lib/form/components/widgets/attach-file/attach-file-cloud-widget.component.ts
Comment (eromano) [inline lib/process-services-cloud/src/lib/form/components/widgets/attach-file/attach-file-cloud-widget.component.ts:154]: can we move what is inside this if in a separate method? try catch block should leave as good practice in a method and not find it in the middle of the code of your method. This is because try-catch is a concern and for the single responsibility principle any method should do only one thing

**Review by eromano — CHANGES_REQUESTED** [review]

File: lib/process-services-cloud/src/lib/form/components/widgets/attach-file/attach-file-cloud-widget.component.ts
Comment (sivakumar414ram) [reply lib/process-services-cloud/src/lib/form/components/widgets/attach-file/attach-file-cloud-widget.component.ts]: Updated. Thanks

File: lib/process-services-cloud/src/lib/form/services/content-cloud-node-selector.service.ts
Comment (sivakumar414ram) [reply lib/process-services-cloud/src/lib/form/services/content-cloud-node-selector.service.ts:68]: I would like to have two method to keep flow understandable. Please do suggest if you have a better name. Thanks

**Comment by arditdomi** [comment]
There is a problem not related to this issue regarding the affected libs. As you see in the travis run no e2e were executed. The following PR contains a fix so we should wait until it gets merged https://github.com/Alfresco/alfresco-ng2-components/pull/6842/files

**Review by arditdomi — APPROVED** [review]

**Pull request merged**
